gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/editor/QuickEditor.kt

```diff
@@ -30,10 +33,19 @@ internal fun QuickEditor(
     ),
 ) {
     val uiState by viewModel.uiState.collectAsState()
+    val aboutEditorViewModel: AboutEditorViewModel = viewModel(
+        factory = AboutEditorViewModelFactory(gravatarQuickEditorParams),
+    )
```

Comment:
I had to move up the creation of the `AboutEditorViewModel` to invoke it on the `done` button tap. The button is in a higher component, and that means we can't control it from the About Editor.